.github/workflows/run-tests.yml

```diff
@@ -6,72 +6,26 @@ on:
   pull_request:
     branches: [ main ]
 
-jobs:
-  build:
 
-    runs-on: ${{ matrix.os }}
-    strategy:
-      fail-fast: false
-      matrix:
-        include:
-        # Test the oldest and newest configuration on Mac
-        - os: macos-latest
-          python-version: 3.8
-          toxenv: py38
-          cache_cmd: -- --dbase-download-dir ~/.chianti
-        - os: macos-latest
-          python-version: '3.10'
-          toxenv: py310
-          cache_cmd: -- --dbase-download-dir ~/.chianti
-        # Test the oldest and newest configuration on Windows
-        - os: windows-latest
-          python-version: 3.8
-          toxenv: py38
-          cache_cmd: -- --dbase-download-dir ~/.chianti
-        - os: windows-latest
-          python-version: '3.10'
-          toxenv: py310
-          cache_cmd: -- --dbase-download-dir ~/.chianti
-        # Test all configurations on Linux
-        - os: ubuntu-latest
-          python-version: 3.8
-          toxenv: py38
-          cache_cmd: -- --dbase-download-dir ~/.chianti
-        - os: ubuntu-latest
-          python-version: 3.9
-          toxenv: py39
-          cache_cmd: -- --dbase-download-dir ~/.chianti
-        - os: ubuntu-latest
+jobs:
+  test:
+    uses: OpenAstronomy/github-actions-workflows/.github/workflows/tox.yml@v1
+    with:
+      posargs: '--dbase-download-dir ~/.chianti'
+      toxdeps: "'tox<4' tox-pypi-filter"
```

Comment:
You probs don't need this

Reply:
Oh I wasn't sure if this was fixed upstream/in the template